<launch>

 <!-- Roscopter Node -->
 <!-- <node ns="apm" pkg="roscopter" type="driver.py" name="roscopter" output="screen" args="--device=/dev/ttyUSB0 --baudrate=57600 --enable-rc-control=true">
  </node>
 -->

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js1" />
    <param name="deadzone" value="0.05" />
  </node>

 <!-- Axes -->
  <param name="axis_roll" value="0" type="int"/>
  <param name="axis_pitch" value="1" type="int"/>
  <param name="axis_throttle" value="2" type="int"/>
  <param name="axis_yaw" value="3" type="int"/>

  <!-- Mode select from the old saitek controller 
  <param name="mode_select13" value="12" type="int"/>
  <param name="mode_select14" value="13" type="int"/>  
  -->

  <!-- Mode select from new saitek controller, on=alt hold, off=stabilize -->
  <param name="mode_select" value="1" type="int"/> <!-- Trigger -->

  <param name="scale_roll" value="-300" type="double"/>
  <param name="scale_pitch" value="-300" type="double"/>
  <param name="scale_throttle" value="-300" type="double"/>
  <param name="scale_yaw" value="300" type="double"/>

  <node pkg="joyToRC" type="joyToRC" name="teleop"/>

</launch>